packages/shared/src/components/cards/PostCardHeader.tsx

```diff
@@ -35,16 +33,13 @@ export const PostCardHeader = ({
   source,
   postLink,
   openNewTab,
-  flagProps,
 }: CardHeaderProps): ReactElement => {
-  const { trending } = flagProps;
   const isFeedPreview = useFeedPreviewMode();
 
   return (
-    <CardHeader className={classNames(className, 'relative')}>
+    <CardHeader className={className}>
```

Comment:
Intended? was it related to trending?

Reply:
Yeah, since the trending flag was absolute positioned this was added, now no longer needed :)